.github/scripts/excluded_from_copyright

```diff
@@ -42,8 +42,6 @@
 ./pkg/assembler/graphql/resolvers/isVulnerability.resolvers.go
 ./pkg/assembler/graphql/generated/hasSLSA.generated.go
 ./pkg/assembler/graphql/resolvers/hasSLSA.resolvers.go
-./pkg/assembler/graphql/generated/isVulnerability.generated.go
-./pkg/assembler/graphql/resolvers/isVulnerability.resolvers.go
```

Comment:
commennt to block merge till we cut a patch version because this breaks API

Reply:
patch version created: https://github.com/guacsec/guac/releases/tag/v0.1.2